demos/security-trifecta/src/pages/login/LoginCard.tsx

```diff
@@ -6,17 +6,66 @@ import {
   IonCardSubtitle,
   IonCardTitle,
   IonIcon,
+  IonSelect,
+  IonSelectOption,
 } from '@ionic/react';
 import { logInOutline } from 'ionicons/icons';
+import { useEffect, useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import { login } from '../../utils/auth';
+import { UnlockMode } from '../../models/UnlockMode';
+import { getUnlockMode, setUnlockMode, canUseLocking } from '../../utils/session-vault';
+import { Device } from '@ionic-enterprise/identity-vault';
 
 const LoginCard: React.FC = () => {
   const history = useHistory();
+  const [displayLockOptions, setLockOptions] = useState<boolean>(false);
+  const [unlockModes, setUnlockModes] = useState<Array<{ mode: UnlockMode; label: string }>>([]);
+  const [selectedMode, setSelectedMode] = useState<UnlockMode | undefined>(undefined);
+
+  useEffect(() => {
+    setLockOptions(canUseLocking);
+    initialize();
+  }, []);
+
+  const initialize = async () => {
+    const newUnlockOptions: { mode: UnlockMode; label: string }[] = [
+      {
+        mode: 'SecureStorage',
+        label: 'Never Lock Session',
+      },
+      {
+        mode: 'CustomPasscode',
+        label: 'Session PIN Unlock',
+      },
+    ];
+
+    Device.isBiometricsEnabled().then((enabled: boolean) => {
+      if (enabled) {
+        newUnlockOptions.push({
+          mode: 'Biometrics',
+          label: 'Biometric Unlock',
+        });
+      }
+
+      setUnlockModes(newUnlockOptions);
+    });
+
+    //initialize to secure storage
+    setSelectedMode(newUnlockOptions[0].mode);
```

Comment:
Instead of relying on the list, this should just explicitly set the mode to `SecureStorage`.

Reply:
Fair enough, done.